src/docs/resources/libraries/README.md

```diff
@@ -41,7 +41,7 @@ The below table lists the different .NET libraries used in Orchard Core:
 | [Serilog.AspNetCore](https://github.com/serilog/serilog-aspnetcore) | Serilog integration for ASP.NET Core. | 7.0.0 | [Apache-2.0](https://github.com/serilog/serilog-aspnetcore/blob/dev/LICENSE) |
 | [Shortcodes](https://github.com/sebastienros/shortcodes) | Shortcodes processor for .NET. | 1.3.3 | [MIT](https://github.com/sebastienros/shortcodes/blob/dev/LICENSE) |
 | [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis) | General purpose redis client. | 2.7.4 | [MIT](https://github.com/StackExchange/StackExchange.Redis/blob/main/LICENSE) |
-| [YesSql](https://github.com/sebastienros/yessql) | .NET document database working on any RDBMS. | 3.5.0 | [MIT](https://github.com/sebastienros/yessql/blob/dev/LICENSE) |
+| [YesSql](https://github.com/sebastienros/yessql) | .NET document database working on any RDBMS. | 4.0.0-beta-0001 | [MIT](https://github.com/sebastienros/yessql/blob/dev/LICENSE) |
```

Comment:
We should probably wait for a release version before merging this, no?

Reply:
We'll have a release before release OC. We need to do testing to ensure everything is good. @sebastienros is going to be doing benchmark too to understand any performance changes

Reply:
And do you want to merge this before that, and then later update to 4.0.0 proper? I'm OK with it, just want to understand.

Reply:
@Piedone 
Yes. We just need people to test it out. We'll release yessql 4 when @jtkech complete the PR that removed Newtonsoft library